### PR TITLE
[💡 FEATURE REQUEST]: Add getter for the `PSR7Worker->httpWorker` property

### PR DESCRIPTION
**Pull request by FluffyDiscord**
| Q             | A
| ------------- | ---
| Bugfix?       | ❌
| Breaks BC?    | ❌
| New feature?  | ✔️

We are able to get the Spiral Worker from `Spiral\RoadRunner\Http\PSR7Worker` but for some reason we cannot get the wrapped `HttpWorker`. This PR fixes that.

## Summary by CodeRabbit

- **New Features**
	- Introduced a method to obtain HTTP worker instances in the application.
	- Added unit tests to ensure availability of the HTTP worker within the `PSR7Worker` class.

### CODE REVIEW

**Review by msmakouz — APPROVED** [review]

**Review by roxblnfk — APPROVED** [review]
Without HttpWorker, we don't have the ability to use some specific functionality like early hints.
I think an access to a lower-level abstraction seems like a good solution. However, I would prefer to see all the capabilities of HttpWorker in PSR7Worker.

**Comment by roxblnfk** [comment]
BTW @FluffyDiscord could you add some Unit Test to cover this getter?

**Review by rustatian — APPROVED** [review]
lgtm

**Comment by FluffyDiscord** [comment]
> BTW @FluffyDiscord could you add some Unit Test to cover this getter?

I guess I can

**Comment by FluffyDiscord** [comment]
Would the last commit do? I am not sure how you would like to create the test, should have asked.

File: tests/Unit/PSR7WorkerTest.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            Worker::create(),
+            $psrFactory,
+            $psrFactory,
+            $psrFactory,
+        );
+
+        ob_end_clean();
+
+        self::assertInstanceOf(HttpWorker::class, $psrWorker->getHttpWorker());
+    }
```
[the remark this answers is not in the document]
Comment (FluffyDiscord) [reply tests/Unit/PSR7WorkerTest.php]: Without `ob_end_clean` PhpUnit throws a warning 
image

**Comment by FluffyDiscord** [comment]
Would it be possible to release new package version with this feature?

**Comment by rustatian** [comment]
Yep, will be released shortly.
@msmakouz Could you please take a look and manage the release?

**Review by sins921 — APPROVED** [review]

**Pull request merged**